docker/app/utils/utils.py

```
@@ -0,0 +1,242 @@
+import torch
+from utils.config import CONFIG
+import nltk
+import os
+import logging
+
+# 評価指標用のダウンロード
+def download_nltk_resources():
+    """必要なnltkリソースをダウンロードします。環境変数でスキップ可能。"""
+    # osモジュールが関数内でローカル変数として宣言される前にアクセスされているので、
+    # このインポートが必要です
+    import os
```

Ruff (0.14.10)

Redefinition of unused `os` from line 4

Remove definition: `os`

(F811)